app/src/main/java/com/keylesspalace/tusky/util/HttpHeaderLink.kt

```diff
@@ -0,0 +1,150 @@
+/* Written in 2017 by Andrew Dawson
+ *
+ * To the extent possible under law, the author(s) have dedicated all copyright and related and
+ * neighboring rights to this software to the public domain worldwide. This software is distributed
+ * without any warranty.
+ *
+ * You should have received a copy of the CC0 Public Domain Dedication along with this software.
```

Comment:
uhm that is not our license

Reply:
It's the license on the most recent version of the code, https://github.com/tuskyapp/Tusky/blob/7a2ee150788ed1e1353214f125c418a9eae6969e/app/src/main/java/com/keylesspalace/tusky/util/HttpHeaderLink.java.

I can change it if you think that's appropriate.

Reply:
Probably an artifact of the time when Vavassor tried to do a license switch, or maybe he copied the code from somewhere else. 
Please change it to 2022 Tusky contributors and GPL